src/components/Input/Input.tsx

```diff
@@ -55,6 +55,10 @@ export type Props = PropsWithSpread<
      * Whether to focus on the input on initial render.
      */
     takeFocus?: boolean;
+    /**
+     * Whether to focus on the input on initial render.
+     */
+    type?: string;
```

Comment:
> Not sure how to make sure checkbox and radio inputs aren't used. Should this component support every other type of input apart from those two?

If you'd like you can define the allowed values here like...
```suggestion
    type?: "text" | "password";
```

Reply:
Thanks, I saw that but wasn't sure which types of input this component should support? as if it's all of them apart from checkbox and input that would be quite a long list

Reply:
This made me think - maybe instead of removing those examples, we should make the `Input` component render checkboxes/radios correctly.

So it would accept all input types, and render a standard `<input>` for them, but if the type is set to checkbox or radio it would use correct CheckboxInput / RadioInput components internally?

Reply:
Yeah this sounds like a better idea. I imagine `<Input type="checkbox/radio">` is used in places and would break if we removed support for it. I'll take a look at this now